index.bs

```diff
@@ -237,41 +296,102 @@ urlPrefix: https://fetch.spec.whatwg.org; spec: FETCH;
             [CEReactions] attribute DOMString importance;
         };
         </pre>
-      <li>TODO: Complete documenting the various points in the link processing model to pass the
-        importance through to the fetch for the referenced link resource.
+      <li><p>We extend [=create a link element request=] of [=Fetching and processing a resource
+        from a link element=] by adding a step before returning the request:</p>
+        <p>Set <var>request</var>'s {{Request/importance}} to the current state of <var ignore=''>
+        el</var>'s <code data-x="attr-link-importance">importance</code> content attribute.</p>
+      <li><p>We extend the [=fetch and process the linked resource=] algorithm of [=modulepreload=]
+        links by adding a step before setting the render-blocking state:</p>
+        <p>Let {{Request/importance}} be the current state of the element's
+        <code data-x="attr-link-importance">importance</code> attribute.</p>
+      <li>We modify the [=List of elements=] to include the 
+        <span data-x="attr-link-importance">importance</span> attribute on the <{link}> element.
     </ol>
 
     <h4 id="script">script</h4>
     <ol>
-      <li>We extend the <{script}> element as follows:
+      <li><p>We add to the <{script}> element content attributes:</p>
+        <p><code data-x="">importance</code> - Importance for [=fetch|fetches=] initiated by the
+          element.
+        </p>
+      <li>We extend the <{script}> element DOM interface as follows:
         <pre class="idl">
         partial interface HTMLScriptElement {
-            [CEReactions] attribute DOMString importance;
+            [CEReactions] attribute DOMString <span data-x="dom-script-importance">importance</script>;
         };
         </pre>
-      <li>TODO: Complete documenting the various points in the script processing model to pass the
-        importance through to the fetch for the referenced script resource.
+      <li><p>We add to the <{img}> element content attribute descriptions:</p>
+        <ol>
+          <li>The <dfn element-attr for="script"><code data-x="attr-script-importance">importance
+            </code></dfn> attribute is an [=importance attribute=]. Its purpose is to set the
+            [=importance=] used when [=fetching=] the script.
+          <li>The <dfn attribute for="HTMLScriptElement"><code
+            data-x="dom-script-importance">importance</code></dfn> IDL attribute must
+            <span>reflect</span> the <code data-x="attr-script-importance">importance</code>
+            content attribute, <span>limited to only known values</span>.
+          <li>We add <code data-x="attr-script-importance">importance</code> to the list of
+            attributes that have no direct effect when changed dynamically.
+        </ol>
+      <li>We modify the list of attributes not to be specified when used in [=data blocks=] to
+        include the <code data-x="attr-script-importance">importance</code> attribute.
```

Comment:
Can you link to this list of attributes?

Reply:
the #data-block link was as close as I could get without using a text-based link. It takes you to the section that includes the list of attributes. It's not linkable directly and doesn't have a number:

![Screen Shot 2022-02-23 at 4 09 45 PM](https://user-images.githubusercontent.com/444165/155408932-f42f44ac-bc8c-4b87-b5d5-2e2699926218.png)
.

Reply:
OK, it might be better to directly link to https://html.spec.whatwg.org/multipage/scripting.html#the-script-element:data-block-2